[2025-06-27T04:29:13.540+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-06-27T04:29:13.606+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.end_task scheduled__2025-04-26T05:00:00+00:00 [queued]>
[2025-06-27T04:29:13.632+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ga4_raw_data_elt_v1.0.end_task scheduled__2025-04-26T05:00:00+00:00 [queued]>
[2025-06-27T04:29:13.634+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-06-27T04:29:13.675+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): end_task> on 2025-04-26 05:00:00+00:00
[2025-06-27T04:29:13.711+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2528) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-27T04:29:13.719+0000] {standard_task_runner.py:72} INFO - Started process 2562 to run task
[2025-06-27T04:29:13.714+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ga4_raw_data_elt_v1.0', 'end_task', 'scheduled__2025-04-26T05:00:00+00:00', '--job-id', '626', '--raw', '--subdir', 'DAGS_FOLDER/GA4_data_elt.py', '--cfg-path', '/tmp/tmpx8fcd2a3']
[2025-06-27T04:29:13.725+0000] {standard_task_runner.py:105} INFO - Job 626: Subtask end_task
[2025-06-27T04:29:13.860+0000] {task_command.py:467} INFO - Running <TaskInstance: ga4_raw_data_elt_v1.0.end_task scheduled__2025-04-26T05:00:00+00:00 [running]> on host f79cc72a155d
[2025-06-27T04:29:14.054+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ga4_raw_data_elt_v1.0' AIRFLOW_CTX_TASK_ID='end_task' AIRFLOW_CTX_EXECUTION_DATE='2025-04-26T05:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-26T05:00:00+00:00'
[2025-06-27T04:29:14.077+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-06-27T04:29:14.105+0000] {logging_mixin.py:190} INFO - end
[2025-06-27T04:29:14.112+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-06-27T04:29:14.145+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-06-27T04:29:14.146+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ga4_raw_data_elt_v1.0, task_id=end_task, run_id=scheduled__2025-04-26T05:00:00+00:00, execution_date=20250426T050000, start_date=20250627T042913, end_date=20250627T042914
[2025-06-27T04:29:14.223+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-06-27T04:29:14.310+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-27T04:29:14.317+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
